### ISSUE TITLE
Upgrade to Kotlin 1.3.60 gives warning WARN: Failed to initialize native filesystem for Windows

### ISSUE THREAD

**Issue by jukkasi** [arturbosch/detekt#2130]
## Expected Behavior
No warnings are shown

## Observed Behavior
Warning ```WARN: Failed to initialize native filesystem for Windows
java.lang.RuntimeException: Could not find installation home path. Please make sure bin/idea.properties is present in the installation directory.
        at org.jetbrains.kotlin.com.intellij.openapi.application.PathManager.getHomePath(PathManager.java:97)
        at org.jetbrains.kotlin.com.intellij.openapi.application.PathManager.getHomePath(PathManager.java:76)
        at org.jetbrains.kotlin.com.intellij.openapi.application.PathManager.findBinFile(PathManager.java:170)
        at org.jetbrains.kotlin.com.intellij.util.loader.NativeLibraryLoader.loadPlatformLibrary(NativeLibraryLoader.java:18)``` in console, task runs successfully though

There is some similar issue in the past https://github.com/arturbosch/detekt/issues/630

## Steps to Reproduce
gradlew detekt

## Your Environment
* Version of detekt used: 1.0.1 or 1.2.0
* Version of Gradle used (if applicable): 5.x
* Operating System and version: Windows 10

**Comment by 3flex** [comment]
This may be fixed by #2129, though I'm a bit confused as to why this has come up again as there's been a workaround for this issue in detekt for a long time now.

Are you able to try with the latest version from `master`? Otherwise v1.2.1 should be released in the next few days with #2129 which hopefully addresses it.

**Comment by 3flex** [comment]
Hi @jukkasi, can you please see if this is still a problem with v1.2.1?
